stable/keel/values.yaml

```diff
@@ -2,9 +2,11 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+namespace: kube-system
```

Comment:
Why add this? Helm has built in support for choosing which namespace to use. I'd recommend removing the `metadata.namespace` keys from the yamls as well, actually.